test/quantization/test_backward_compatibility.py

```diff
@@ -0,0 +1,126 @@
+                                 "serialized",
+                                 munged_id)
+
+        subname_output = ""
+        if subname:
+            base_name += "_" + subname
+            subname_output = " ({})".format(subname)
+
+        input_file = base_name + ".input.pt"
+        state_dict_file = base_name + ".state_dict.pt"
+        scripted_module_file = base_name + ".scripted.pt"
+        traced_module_file = base_name + ".traced.pt"
+        expected_file = base_name + ".expected.pt"
+
+        # saving code : uncomment when adding a new test
+        # and run the new test, e.g.
+        # python test/quantization/test_backward_compatibility.py TestSerialization.test_conv3d
+        # data = ...
+        # data = torch.quantize_per_tensor(data, 0.5, 2, torch.quint8)
+        # torch.save(data, input_file)
+        # torch.save(qmodule.state_dict(), state_dict_file)
+        # torch.jit.save(torch.jit.script(qmodule), scripted_module_file)
+        # torch.jit.save(torch.jit.trace(qmodule, data), traced_module_file)
+        # torch.save(qmodule(data), expected_file)
+
+        data = torch.load(input_file)
+        qmodule.load_state_dict(torch.load(state_dict_file))
+        qmodule_scripted = torch.jit.load(scripted_module_file)
+        qmodule_traced = torch.jit.load(traced_module_file)
+
+        expected = torch.load(expected_file)
+        self.assertEqual(qmodule(data), expected, prec=prec)
+        self.assertEqual(qmodule_scripted(data), expected, prec=prec)
+        self.assertEqual(qmodule_traced(data), expected, prec=prec)
+
+    @unittest.skipUnless(
+        'fbgemm' in torch.backends.quantized.supported_engines,
+        " Quantized operations require FBGEMM. FBGEMM is only optimized for CPUs"
+        " with instruction set support avx2 or newer.",
+    )
+    def test_conv2d(self):
+        # quantized conv module
+        qconv = nnq.Conv2d(3, 3, kernel_size=3, stride=1, padding=0, dilation=1,
+                           groups=1, bias=True, padding_mode="zeros")
+        self._test_op(qconv)
+        # TODO: graph mode quantized conv2d module
+
+    @unittest.skipUnless(
+        'fbgemm' in torch.backends.quantized.supported_engines,
+        " Quantized operations require FBGEMM. FBGEMM is only optimized for CPUs"
+        " with instruction set support avx2 or newer.",
+    )
+    def test_conv2d_relu(self):
+        # quantized conv module
+        qconv = nniq.ConvReLU2d(3, 3, kernel_size=3, stride=1, padding=0, dilation=1,
+                                groups=1, bias=True, padding_mode="zeros")
+        self._test_op(qconv)
+        # TODO: graph mode quantized conv2d module
+
+    @unittest.skipUnless(
+        'fbgemm' in torch.backends.quantized.supported_engines,
+        " Quantized operations require FBGEMM. FBGEMM is only optimized for CPUs"
+        " with instruction set support avx2 or newer.",
+    )
+    def test_conv3d(self):
```

Comment:
Can you also add a test for linear and linear_relu?

Reply:
sure